api/src/main/java/org/openmrs/module/patientflags/api/FlagService.java

```diff
@@ -201,6 +248,14 @@
 	 */
 	@Authorized( { PatientFlagsConstants.PRIV_MANAGE_FLAGS })
 	public void purgeTag(Integer tagId);

```

Comment:
You need to have at least a line between methods.

Reply:
Done